actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -16,14 +16,13 @@ module ActionDispatch # :nodoc:
   # should never be used directly in controllers. Controllers should use the
   # methods defined in ActionController::Base instead. For example, if you want
   # to set the HTTP response's content MIME type, then use
-  # ActionControllerBase#headers instead of Response#headers.
+  # ActionController::Metal#headers instead of Response#headers.
```

Comment:
I'm not sure what to do about this one. `ActionController::Metal` is where the `headers` method is defined, but it's a `delegate` so even this change isn't creating a link to anything

https://github.com/rails/rails/blob/3bcd48f6d8b0d87e02b75d124984895cf28e6f9c/actionpack/lib/action_controller/metal.rb#L147-L148

Reply:
Ah cool, that sounds like what I was looking for. I'll split it all into another PR and keep this one simple